controllers/kubemarkmachine_controller.go

```diff
@@ -409,7 +409,8 @@ func (r *KubemarkMachineReconciler) Reconcile(ctx context.Context, req ctrl.Requ
 		}
 	}
 
-	machine.Spec.ProviderID = pointer.StringPtr(fmt.Sprintf("kubemark://%s", kubemarkMachine.Name))
+	machine.Spec.ProviderID = pointer.String(fmt.Sprintf("kubemark://%s", kubemarkMachine.Name))
+	kubemarkMachine.Spec.ProviderID = pointer.String(fmt.Sprintf("kubemark://%s", kubemarkMachine.Name))
```

Comment:
any reason not to do the `fmt.Sprintf` once and assign to a var?

Reply:
Good idea - updated.